packages/next/src/server/config-shared.ts

```diff
@@ -1079,6 +1067,12 @@ export interface NextConfig extends Record<string, any> {
   watchOptions?: {
     pollIntervalMs?: number
   }
+
+  /**
+   * User agent of bots that can't handle streaming metadata.
```

Comment:
sorry I edited this one after I submitted the last comment 🙈 

lgtm after

```suggestion
   * User agent of bots that should opt-out streaming metadata.
```